submissions/pozzitive11/popup/css/style.css

```diff
@@ -94,6 +94,10 @@ li {
   -moz-appearance: none;
   appearance: none;
   position: absolute;
+  top: 0;
+  left: 0;
+  cursor: pointer;
+  outline: none;
```

Comment:
On Google Chrome you can't see it but Firefox shows all) Check this video and see what's wrong https://share.vidyard.com/watch/yLXJRZFy2TUJCFUWRMtnzd?